src/wrapper.rs

```diff
@@ -119,18 +119,17 @@
 ///
 /// ### Object
 ///
-/// Objects -- classes and interfaces.  Note that the class name, if
-/// available, must be specified after the $foreign type; see below
-/// for [non-derivable classes][#non-derivable-classes].
+/// Objects -- classes.  Note that the class name, if available, must be specified after the
```

Comment:
Two whitespaces in `classes.  Note`. Is it expected? Otherwise please remove one or start a new paragraph.